### ISSUE TITLE
Getting java.lang.NoSuchMethodError: for 'io.netty.util.internal.ReflectionUtil.trySetAccessible(java.lang.reflect.AccessibleObject)'

### ISSUE THREAD

**Issue by himanshu2705** [jchambers/pushy#866]
Hi,

I have added following dependencies

<dependency>
			<groupId>com.eatthepath</groupId>
			<artifactId>pushy</artifactId>
			<version>0.14.2</version>
		</dependency>

and

<dependency>
			<groupId>io.netty</groupId>
			<artifactId>netty-all</artifactId>
			<version>4.1.60.Final</version>
			<scope>runtime</scope>
		</dependency>

I am getting following error on deploying my project

```
Caused by: java.lang.NoSuchMethodError: 'java.lang.Throwable io.netty.util.internal.ReflectionUtil.trySetAccessible(java.lang.reflect.AccessibleObject)'
	at io.netty.channel.nio.NioEventLoop$5.run(NioEventLoop.java:199)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at io.netty.channel.nio.NioEventLoop.openSelector(NioEventLoop.java:192)
	at io.netty.channel.nio.NioEventLoop.<init>(NioEventLoop.java:149)
	at io.netty.channel.nio.NioEventLoopGroup.newChild(NioEventLoopGroup.java:127)
	at io.netty.channel.nio.NioEventLoopGroup.newChild(NioEventLoopGroup.java:36)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:84)
	at io.netty.util.concurrent.MultithreadEventExecutorGroup.<init>(MultithreadEventExecutorGroup.java:58)
	at io.netty.channel.MultithreadEventLoopGroup.<init>(MultithreadEventLoopGroup.java:51)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:87)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:82)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:63)
	at io.netty.channel.nio.NioEventLoopGroup.<init>(NioEventLoopGroup.java:51)
	at com.eatthepath.pushy.apns.ApnsClient.<init>(ApnsClient.java:141)
	at com.eatthepath.pushy.apns.ApnsClientBuilder.build(ApnsClientBuilder.java:564)
	at com.dronzup.core.service.impl.ApnsPushNotificationServiceImplV2.customInit(ApnsPushNotificationServiceImplV2.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:366)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:311)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:134)
	... 88 more
```

I checked the downloaded files there is a class io.netty.util.internal.ReflectionUtil having method signature

public static Throwable trySetAccessible(AccessibleObject object, boolean checkAccessible) 

It does not have the method signature as shown in error.

What should I do to remove this error?
Thanks.
![Screenshot from 2021-03-23 17-50-41](https://user-images.githubusercontent.com/81236013/112146140-395a3080-8c01-11eb-91e0-47fc8b884c3d.png)

**Comment by jchambers** [comment]
Hello!

This sounds more like a request for general support than it does like a bug report or a feature request. We generally ask people to move these kinds of questions to [the Pushy mailing list](https://groups.google.com/d/forum/pushy-apns) because:

1. The mailing list has a wider audience, and you're more likely to find somebody who has experience with your particular circumstances on the mailing list.
2. Posting your question to the mailing list can help other people who are having the same problem.
3. Because there isn't a bug to fix or a feature to add, we can't reasonably solve your problem by making any kind of engineering change.

I emphasize that we'd be happy to help, but let's move this conversation to a different venue.

That said, this usually happens to folks who have multiple versions of Netty on their classpath (see #844, for example). I'd guess you probably have a Netty version conflict with another one of your dependencies. I'm going to close this issue for now since I don't think there's anything we can change within Pushy to address the situation, but again, please don't hesitate to ask for help on the mailing list!

**Comment by jchambers** [comment]
Also see #863 for an even more recent example.

**Comment by himanshu2705** [comment]
Thanks jchambers, will use Pushy mailing list for further such type of problems.

By the way I found the fix of the issue, earlier I was using com.relayrides.pushy for APNS push notifications, I have not removed its dependency which was causing the issue, and it got fixed when i removed com.relayrides.pushy dependency. 

We have to migrate to Apple HTTP 2 push api so we moved to com.eatthepath.pushy

Will not ask this question in mailing list as it has been solved, but I am facing different problem now, so will use mailing list for that.

Thanks.
